integration/env.go

```diff
@@ -24,18 +25,40 @@ import (
 	"github.com/ignite/cli/ignite/pkg/cmdrunner/step"
 	"github.com/ignite/cli/ignite/pkg/cosmosfaucet"
 	"github.com/ignite/cli/ignite/pkg/gocmd"
+	"github.com/ignite/cli/ignite/pkg/gomodulepath"
 	"github.com/ignite/cli/ignite/pkg/httpstatuschecker"
 	"github.com/ignite/cli/ignite/pkg/xexec"
 	"github.com/ignite/cli/ignite/pkg/xurl"
 )
 
 const (
 	ServeTimeout = time.Minute * 15
-	IgniteApp    = "ignite"
 	ConfigYML    = "config.yml"
 )
 
-var isCI, _ = strconv.ParseBool(os.Getenv("CI"))
+var (
+	isCI, _   = strconv.ParseBool(os.Getenv("CI"))
+	IgniteApp string
+)
+
+func init() {
```

Comment:
Can we avoid using init, maybe not that important for this case but I usually find it causing side effects.

I propose we:

- Create a package level func that does the installation, named as `InstallBinary()` with `...Options`s if necessary. 
- Store a `sync.Once` in a global variable: `var installBinaryOnce sync.Once` 
- And add logic into `New` to call `installBinaryOnce` with `InstallBinary`.

Reply:
Done here [e8dcbdb](https://github.com/ignite/cli/pull/2639/commits/e8dcbdb3b2d2f31b74747618852893eebf52930b)